build.assets/Dockerfile

```diff
@@ -42,17 +51,12 @@ RUN (set -ex && mkdir -p /gopath/src/github.com/gravitational && \
 RUN (set -ex && mkdir -p /opt/protoc && \
      wget --quiet -O /tmp/${TARBALL} ${PROTOC_URL} && \
      unzip -d /opt/protoc /tmp/${TARBALL} && \
-     mkdir -p /src/github.com/gogo/ /src/github.com/grpc-ecosystem && \
-     git clone https://github.com/gogo/protobuf --branch ${GOGO_PROTO_TAG} /src/github.com/gogo/protobuf && cd /src/github.com/gogo/protobuf && make install && \
-     git clone https://github.com/grpc-ecosystem/grpc-gateway --branch ${GRPC_GATEWAY_TAG} /src/github.com/grpc-ecosystem/grpc-gateway && cd /src/github.com/grpc-ecosystem/grpc-gateway && pwd && go install ./protoc-gen-grpc-gateway)
+     mkdir -p /gopath/src/github.com/gogo/ /gopath/src/github.com/grpc-ecosystem && \
+     git clone https://github.com/gogo/protobuf --branch ${GOGO_PROTO_TAG} /gopath/src/github.com/gogo/protobuf && \
+     cd /gopath/src/github.com/gogo/protobuf && make install && \
+     git clone https://github.com/grpc-ecosystem/grpc-gateway --branch ${GRPC_GATEWAY_TAG} /gopath/src/github.com/grpc-ecosystem/grpc-gateway && \
+     cd /gopath/src/github.com/grpc-ecosystem/grpc-gateway && GO111MODULE=on go install ./protoc-gen-grpc-gateway)
```

Comment:
`GO111MODULE=on` added for consistent behavior with when `go install` is run from `/src` (or anywhere outside `GOPATH`)